ci/test-stable.sh

```diff
@@ -31,6 +31,14 @@ testName=$(basename "$0" .sh)
 echo "Executing $testName"
 case $testName in
 test-stable)
+  ci/affects-files.sh \
```

Comment:
Let's promote this check to right up top, around line 12.  We want to run nothing from here the book/ is updated (test-checks has already done it all)